### ISSUE TITLE
Handle .mem files in sim projects

### ISSUE THREAD

**Issue by dinyar** [ipbus/ipbb#144]
When using XPM modules I need to copy the .mem files that hold the contents of the memories into the project folder (i.e. `proj/[project name]`) for the sim project. Apparently Vivado does this automatically for itself once one adds mem files to a Vivado project. 

It might make sense to extend ipbb to understand the .mem extension for simulation projects and when added copy those to the project folder. Currently I link the mem files over into the project folder as part of the CI, but in the upcoming common CI we'd prefer to have all project specific tasks contained in ipbb config files.

**Comment by alessandrothea** [comment]
Yep, it would be good for `ipbb` to take care of that. This is in fact a similar problem to the `mifs` and in fact, it would be good to have consistent handling of these "memory data files". Would you be able to put together a minimal example?

**Comment by dinyar** [comment]
Hi,

Sorry for the delay, a minimal example can be found at https://gitlab.cern.ch/dinyar/mem-files-in-ipbb, you should have access, let me know if not.

Cheers,
Dinyar

**Comment by dinyar** [comment]
Hi @alessandrothea,

Any news on this? If you're fine with it I could try to give this a shot and make a PR. Let me know.

Cheers,
Dinyar
